### PR TITLE
Update Index.html

### PR DESCRIPTION
**Pull request by medubelko**
Content and code updates to index.html.

For now, both the carousel and the image cascade are in place. I'm open to including one or the other, or both.

Preview at: `/home/michaeldu/dev/gafferHQBuilds/indexUpdate`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,16 +15,16 @@
...
-                <h3 class="h4">About Gaffer</h3>
-                <div class="spacer-15"></div>
-                <p class="lead">Gaffer is a general purpose node-based application designed for use in the visual effects industry.</p>
-		<p>It provides basic tools for procedural scene generation, shader authoring, rendering, and image compositing.</p>
+        <div class="row">
+            <div class="col-md-10 col-md-offset-1">
+                <h1 class="text-center landing">Introducing Gaffer</h1>
+                <p class="lead">Gaffer is a free, open-source, node-based VFX application that enables look developers, lighters, and compositors to easily build, tweak, iterate, and render scenes. Built with flexibility in mind, Gaffer supports in-application scripting in Python and OSL, so VFX artists and technical directors can design shaders, automate processes, and build production workflows.</p>
+                <p class="lead">With hooks in both C++ and Python, Gaffer's readibly extensible API provides both professional studios and enthusiasts with the tools to add their own custom modules, nodes, and UI.</p>
```
Comment (andrewkaufman) [inline index.html]: "readibly" -> "readily"

File: index.html [hunk 2]
```diff
@@ -29,7 +29,46 @@ <h1 class="text-center landing">Introducing Gaffer</h1>
...
+    <div class="row ptb-15 wow">
+        <div class="col-xs-12 col-md-6">
+            <!-- Lost in Space S1 forest robot grey -->
+            <a class="cbox-gallary1" href="http://image-engine.com/wp-content/uploads/2018/04/101_040_600_AO.1006.jpg">
+                <div class="bg-img" style="height: 300px; background-image: url(http://image-engine.com/wp-content/uploads/2018/04/101_040_600_AO.1006.jpg);"></div>
+            </a>
+        </div>
+        <div class="col-xs-12 col-md-6">
+            <h3 class="mt-60 text-center">Create Scenes</h3>
+            <p>Import, build, and manipulate scenes non-destructively. Duplicate, combine, join, prune, split, and inspect scenes of arbitrary complexity. Iterate over your sequence's components in a fast and frustration-free interface.</p>
```
Comment (andrewkaufman) [inline index.html]: "frustration-free interface" ... that's a bold claim...

File: index.html [hunk 3]
```diff
@@ -29,7 +29,46 @@ <h1 class="text-center landing">Introducing Gaffer</h1>
...
+    <div class="row ptb-15 wow">
+        <div class="col-xs-12 col-md-6 pull-right">
+            <!-- Shader graph -->
+            <a class="cbox-gallary1" href="img/headers/header_graph.jpg">
+                <div class="bg-img" style="height: 300px; background-image: url(img/headers/header_graph.jpg); background-position: right"></div>
+            </a>
+        </div>
+        <div class="col-xs-12 col-md-6">
+            <h3 class="mt-60 text-center">Build Shaders</h3>
+            <p>With full support for <a href="https://www.solidangle.com/arnold" alt="Arnold homepage">Arnold</a> and <a href="http://www.3delight.com" alt="3Delight homepage">3Delight's</a> powerful industry-standard shaders, use the robust node plug-outs to combine and design. For extra customization, arrange and write custom shaders using <a href="http://opensource.imageworks.com/?p=osl" alt="Open Shading Language homepage">Open Shading Language</a> nodes and code.</p>
```
Comment (andrewkaufman) [inline index.html]: "use the robust node plug-outs to combine and design." I don't understand this statement

File: index.html [hunk 4]
```diff
@@ -29,7 +29,46 @@ <h1 class="text-center landing">Introducing Gaffer</h1>
...
+    <div class="row ptb-15 wow">
+        <div class="col-xs-12 col-md-6">
+            <!-- Gaffy IPR -->
+            <a class="cbox-gallary1" href="img/headers/carousel_bot.jpg">
+                <div class="bg-img text-center" style="height: 300px; background-image: url(img/headers/carousel_bot.jpg); background-position: left"></div>
+            </a>
+        </div>
+        <div class="col-xs-12 col-md-6">
+            <h3 class="mt-60 text-center">Design Workflows</h3>
+            <p class="">Whether for 3D sequences, 2D image processing, basic compositing, or pipeline processes, design workflows all within a single tool. Use logic, filters, and expressions to make scalable node networks for automated and responsive VFX solutions.</p>
```
Comment (andrewkaufman) [inline index.html]: 2D image processing and basic compositing are kind of the same thing. Only mention one or the other.

**Comment by andrewkaufman** [comment]
Other than my minor comments, this is fine by me, but I think @johnhaddon should have a glance at the new copy before we merge.

**Comment by medubelko** [comment]
Incorporated Andrew's suggestions.

**Comment by johnhaddon** [comment]
Thanks Michael!

**Pull request merged**